ycms/cms/templates/zentral.html

```diff
@@ -0,0 +1,8 @@
+{% extends "_base.html" %}
+{% block content %}
+<ul>
+    {% for ward in total_wards %}
```

Comment:
```suggestion
    {% for ward in wards %}
```

`total_wards` is what you called your variable in `zentral.py`. However, you assigned that variable to `"wards"` in the context object, so we have to use that here :)